packages/core/src/App/Containers/CloseMXAccountModal/close-mx-account-modal.jsx

```diff
@@ -6,89 +6,121 @@ import PropTypes from 'prop-types';
+            );
+        } else if (is_eu) {
+            return (
+                <Localize i18n_default_text='You are no longer able to trade digital options on any of our platforms. Also, you can’t make deposits into your Options account.' />
+            );
+        }
+        return (
+            <Localize i18n_default_text='You are no longer able to trade digital options on any of our platforms. Also, you can’t make deposits into your account.' />
+        );
+    };
+    const OptionsAccountClosedText = () => {
+        if (is_uk) {
+            return (
+                <Localize i18n_default_text='You will lose access to your Gaming account when it gets closed, so be sure to withdraw all your funds. (If you have a CFDs account, you can also transfer the funds from your Gaming account to your CFDs account.)' />
+            );
+        } else if (is_eu) {
+            return (
+                <Localize i18n_default_text='You will lose access to your Options account when it gets closed, so be sure to withdraw all your funds. (If you have a CFDs account, you can also transfer the funds from your Options account to your CFDs account.)' />
+            );
+        }
+        return (
+            <Localize i18n_default_text='You will lose access to your account when it gets closed, so be sure to withdraw all your funds.' />
+        );
+    };
+
+    return (
+        <>
+            <div className='close-mx-account-header'>
+                <Icon icon='IcAmplifier' />
+                <Text as='h1' weight='bold' className='close-mx-account-header__heading' align='center'>
+                    <HeaderText />
+                </Text>
+            </div>
+            <div className='close-mx-account__content'>
+                <Text as='p' size='s'>
+                    <ProductLineupText />
+                </Text>
+                <br />
+                <Text align='left' weight='bold' as='p' size='s'>
+                    <Localize i18n_default_text='What this means for you' />
+                </Text>
+                <Text as='p' size='s'>
+                    <NoDigitalOptionsText />
+                </Text>
+                <br />
+                <Text align='left' weight='bold' as='p' size='s'>
+                    <Localize i18n_default_text='Withdraw all your funds' />
+                </Text>
+                <Text align='left' as='p' size='s'>
+                    <OptionsAccountClosedText />
+                </Text>
+            </div>
+            <div className='close-mx-account dc-dialog__footer'>
+                <Button
+                    className='continue-button'
+                    onClick={() => {
+                        showCloseMXAccountPopup(false);
+                        localStorage.setItem('hide_close_mx_account_notification', '1');
+                        removeNotificationMessageByKey({ key: 'close_mx_account' });
```

Comment:
```suggestion
                        localStorage.setItem('hide_close_mx_mlt_account_notification', '1');
                        removeNotificationMessageByKey({ key: 'close_mx_mlt_account' });
```

Reply:
(throughout) (suggestion) let's use a better naming for the flags and function, since we are expanding its ability.

Reply:
Good idea, I will change this.